### DISCUSSION TITLE
Is WezTerm controllable from the command-line?

### DISCUSSION THREAD

**Discussion by alexherbo2** [wezterm/wezterm#538] [Q&A]
I haven’t found a way to control WezTerm from the command-line.

I know about `wezterm start -- <command> [arguments]` to open a new terminal, but I haven’t found a way to `SplitVertical={domain="CurrentPaneDomain"}` as documented in the [default shortcuts] page.

The use case is to add terminal window and split commands to [Kakoune], which doesn’t handle the windowing itself.

[Default shortcuts]: https://wezfurlong.org/wezterm/config/keys.html#default-shortcut--key-binding-assignments
[Kakoune]: https://kakoune.org

**Comment by wez** [comment]
Take a look at the `cli` subcommand:

```
; wezterm cli split-pane --help
wezterm-cli-split-pane 0.1.0
split the current pane.
Outputs the pane-id for the newly created pane on success

USAGE:
    wezterm cli split-pane [FLAGS] [OPTIONS] [prog]...

FLAGS:
    -h, --help          Prints help information
        --horizontal    Split horizontally rather than vertically
    -V, --version       Prints version information

OPTIONS:
        --cwd <cwd>            Specify the current working directory for the initially spawned program
        --pane-id <pane-id>    Specify the pane that should be split. The default is to use the current pane based on
                               the environment variable WEZTERM_PANE

ARGS:
    <prog>...    Instead of executing your shell, run PROG. For example: `wezterm start -- bash -l` will spawn bash
                 as if it were a login shell
```
